src/components/Navigation.js

```diff
@@ -1,18 +1,26 @@
+import './Navigation.css';
 import { NavLink } from 'react-router-dom';
 
 function Navigation() {
   return (
     <nav className="navbar">
-      <h2>Bookstore CMS</h2>
+      <div className="navbar-items">
+        <h1 className="primary-header">Bookstore CMS</h1>
+        <ul>
+          <li>
+            <NavLink to="/">Books</NavLink>
+          </li>
+          <li>
+            <NavLink to="catagories">catagories</NavLink>
+          </li>
```

Comment:
- [ ] Please make your navigation text uppercased to match the actual design

yours|actual
---|---
![image](https://github.com/mahdinoori2000/bookstore/assets/91486791/8f8218e2-a741-48ab-b92e-ed812d606fa5)|![image](https://github.com/mahdinoori2000/bookstore/assets/91486791/a22bb68a-f48a-431d-9977-18b6ca931225)